### ISSUE TITLE
Other El Niño regions besides 3.4

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#400]
Although we allow a user to select `nino3` and `nino4` in addition to the default `nino3.4`, all output (including figure titles) assume that 3.4 was selected regardless of the user's actual choice.

It should be easy to fix this by having a dictionary with these 3 region names as keys and the full name of each region as a value.

**Comment by xylar** [comment]
A note to myself:  While cleaning up indexNino3.4, might as well fix this:
```
   if dataSource == 'HADIsst':
       dataPath = "{}/HADIsst_nino34.nc".format(observationsDirectory)
       obsTitle = 'HADSST'
   else:
       dataPath = "{}/ERS_SSTv4_nino34.nc".format(observationsDirectory)
       obsTitle = 'ERS SSTv4'
```
Should have an `elif dataSorce == 'ERS SSTv4':` and a final `else:` with some kind of error message.  Otherwise, a typo in `HadIsst` would unexpectedly produce `ERS SSTv4` output.

**Comment by xylar** [comment]
Fixed in #408
